mix.exs

```diff
@@ -38,7 +38,7 @@ defmodule Mana.MixProject do
   defp deps do
     [
       {:ex_doc, "~> 0.19", only: :dev, runtime: false},
-      {:dialyxir, "~> 1.0.0-rc.3", only: [:dev, :test], runtime: false},
```

Comment:
Any reason why this change is in here? I think Dialyxir's new warnings are far better.

Reply:
If you downgraded to get the output to put into the ignore file, on the newer version you can run `mix dialyzer --format dialyzer` to get the old format.

Reply:
yep, exactly that was a problem for me. But I returned the new version